infra-templates/nfs/4/rancher-compose.yml

```diff
@@ -1,32 +1,43 @@
 .catalog:
   name: "Rancher NFS"
-  version: "0.3.0"
+  version: 0.4.0
   description: |
     Docker volume plugin for NFS
-  minimum_rancher_version: v1.6.0
+  minimum_rancher_version: v1.4.0-rc1
   questions:
   - variable: "NFS_SERVER"
-    description: "IP or hostname of the default NFS Server"
+    description: "IP or hostname of the NFS Server"
     label: "NFS Server"
     required: true
     type: "string"
   - variable: "MOUNT_DIR"
-    label: "Export Base Directory"
-    description: "The default exported base directory"
+    label: "Mount Directory"
+    description: "The directory on the NFS server to mount"
     type: "string"
     required: true
   - variable: "MOUNT_OPTS"
     label: "Mount Options"
-    description: "Comma delimited list of default mount options, for example: 'proto=udp'. Do not specify `nfsvers` option, it will be ignored."
+    description: "Comma delimited list of mount options, for example: 'proto=udp'. Do not specify `nfsvers` option, it will be ignored."
     type: "string"
   - variable: "NFS_VERS"
     label: NFS Version
-    description: The NFS version to use
+    description: Default NFS version to use
     type: enum
     required: true
     default: nfsvers=4
     options:
     - nfsvers=4
+    - nfsvers=3
+    - nfsvers=2
```

Comment:
Does this actually work with NFS v2? If so, that's fine. Otherwise we should remove it from this list.